ride_to_work_by_bike_config.toml

```diff
@@ -21,3 +21,5 @@ facebookUrl = "https://www.facebook.com/spolekautomat"
 instagramUrl = "https://www.instagram.com/spolekautomat"
 twitterUrl = "https://twitter.com/spolekautomat"
 youtubeUrl = "https://www.youtube.com/@spolekautomat"
+urlAppStore = "https://apps.apple.com/cz/app/do-pr%C3%A1ce-na-kole/id1564442780"
```

Comment:
Perhaps you could write a unit test that tests existence (request/response) of these URLs.

Reply:
Good idea.
I used the [request](https://docs.cypress.io/api/commands/request) command which seems to be more suitable for this purpose. `visit` is primarily designed for navigating to internal urls.

Reply:
> Good idea. I used the [request](https://docs.cypress.io/api/commands/request) command which seems to be more suitable for this purpose. 

Yes, you are right, the `cy.request()` method is better for this use case.

>`visit` is primarily designed for navigating to internal urls.

`cy.visit()` method is better to use in e2e tests, but according  method doc use case is  _Visit a remote URL_.